packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -104,10 +114,6 @@ export const getStyles = (props: ICheckboxStyleProps): ICheckboxStyles => {
                 background: checkboxBackgroundCheckedHovered,
                 borderColor: checkboxBorderColorCheckedHovered,
               },
-              [`.${classNames.checkbox}`]: {

```

Comment:
this block is not needed as it's already defined below within `checkbox` section